app/webserver/helpers.go

```diff
@@ -240,36 +240,42 @@ func joinGroup(joinGroupmessage JoinGroupMessage) *store.Session {
 	log.Infoln("[axolotl] joinGroup", joinGroupmessage.ID)
 	group, err := textsecure.JoinGroup(joinGroupmessage.ID)
 	if err != nil {
+		log.Errorln("[axolotl] joinGroup failed", err)
 		// update group also in the case that the group is already joined to remove the join message
-		if group != nil {
-			session, _ := store.SessionsModel.GetByUUID(group.Hexid)
-			session.Name = group.DecryptedGroup.Title
-			session.GroupJoinStatus = group.JoinStatus
-			store.UpdateSession(session)
+		if group == nil {
+			return nil
 		}
-		log.Errorln("[axolotl] joinGroup", err)
-		ShowError(err.Error())
-		return nil
+
 	}
+
 	log.Infoln("[axolotl] joining group was succesful", group.Hexid)
```

Comment:
I'm not yet happy with this, because in the case that `err != nil && group != nil` the logs look strange
```
[axolotl] joinGroup failed
[axolotl] joining group was succesful
```

Reply:
There is a typo in "Successful" btw ;)

Reply:
Now the log would show:
```
warn [axolotl] error while joining group {some_errormessage}
info [axolotl] joining group was successful {some_id}
```
This would indicate that there was an error, but that joining was successful nevertheless.

Do you agree that this is better?

If group would be nil, too the log would show:
```
warn [axolotl] error while joining group {some_errormessage}
error [axolotl] joinGroup failed
```

Reply:
The group join was only successful, when `group.JoinStatus == store.GroupJoinStatusJoined` is `true`, maybe we can put it in an if.

Reply:
I moved the line a bit down into the if that was already there. :)